conceptnet5/builders/morphology.py

```diff
@@ -0,0 +1,57 @@
+import click
```

Comment:
Click is never used. The import statements should be sorted (here and ex. in `sparse_matrix_builder.py`).